modules/web/src/app/settings/admin/nav/template.html

```diff
@@ -106,6 +106,15 @@
         </a>
       </div>
 
+      <div class="sub-menu-item"
```

Comment:
We should also add this entry for collapsed side nav.

Reply:
Good point, sorry I completely overlooked it.